app/models/setting/content.rb

```diff
@@ -104,7 +104,7 @@ def self.default_settings
       self.set('pulp_docker_registry_port', N_("The port used by Pulp Crane to provide Docker Registries"),
                5000, N_('Pulp Docker registry port')),
       self.set('pulp_export_destination', N_("On-disk location for exported repositories"),
-               "/var/lib/pulp/katello-export", N_('Pulp export destination filepath')),
+               "/var/lib/pulp/exports", N_('Pulp export destination filepath')),
```

Comment:
are there selinux implications to changing this for pulp2 installs?

Reply:
or permission/installer/packaging implications?

Reply:
hmm are you saying we should leave this alone for this PR and have the installer determine the settings (based on pulp3/2) ?

Reply:
We discussed further and decided to have a new setting for `pulpcore_export_directory` because the permissions are quite  different between `/var/lib/pulp/katello-exports` and `/var/lib/pulp/exports`

Reply:
+1 to this!

Reply:
updated.